landing_page/app/views/microposts/show.html.erb

```diff
@@ -0,0 +1,31 @@
+<p id="notice"><%= notice %></p>
+
+<p>
```

Comment:
A `<div>` tag may be more appropriate here. I believe `<p>` tags are mainly supposed to be used for paragraphs of text, not to contain headers.